content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -169,6 +171,27 @@ CREATE TABLE "Example" (
 CREATE INDEX "Example_val_idx" ON "Example" USING HASH (val);
 ```
 
+
+
+### Configuring the access type of indexes with `type` (PostgreSQL)
```

Comment:
Is this suppose to be PostgreSQL/ SQL Server?

Reply:
This is a copy/paste error, I forgot to adapt the headline completely.